### PR TITLE
Topic recognition 45542976 zhengzhgao li

### PR DESCRIPTION
**Pull request by zhengzhao610**
my topic is brain segmatation,  due to the laptop issue all the log commit in same day

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Some commit messages should be more specific
- Cannot locate main training loop for training, `test.py` is does not have valid imports, results are hence questionable.
- Code INVALID for Easy Difficulty

Other Feedback
- I would not use "1" as the folder name
- Please group the `.png` files and put them in a sub-directory
- Please follow python file  and folder naming convention e.g. `lorem_ipsum_123_abc.py`
- Many images not showing

**IMPOTANT**:  YOU MUST JUSTIFY YOUR RESULTS via a reply.  I need to know how exactly your code works and how the results came to be.

**Comment by zhengzhao610** [comment]
hi，
due to the laptop issue, I have been unable to upload to GitHub before and I was in a hurry when submitting the file at the end, so some file may be lost , and I wrote it in jupyter notebook at the beginning and then moved to visual studio code, so the Unet and test files are imported same thing


获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: SiyuLiu0329 <notifications@github.com>
发送时间: Sunday, November 22, 2020 5:23:29 PM
收件人: shakes76/PatternFlow <PatternFlow@noreply.github.com>
抄送: Zhengzhao Li <zhengzhao.li@uq.net.au>; Author <author@noreply.github.com>
主题: Re: [shakes76/PatternFlow] Topic recognition 45542976 zhengzhgao li (#148)


  *   Some commit messages should be more specific
  *   Cannot locate main training loop for training, test.py is does not have valid imports, results are hence questionable.
  *   Code INVALID for Easy Difficulty

Other Feedback

  *   I would not use "1" as the folder name
  *   Please group the .png files and put them in a sub-directory
  *   Please follow python file and folder naming convention e.g. lorem_ipsum_123_abc.py
  *   Many images not showing

IMPOTANT: YOU MUST JUSTIFY YOUR RESULTS via a reply. I need to know how exactly your code works and how the results came to be.

―

**Comment by zhengzhao610** [comment]
I try to send you the images that I uploaded but it size too big can’t send you through the mail [Image.jpeg] [Image.jpeg]

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: SiyuLiu0329 <notifications@github.com>
发送时间: Sunday, November 22, 2020 5:23:29 PM
收件人: shakes76/PatternFlow <PatternFlow@noreply.github.com>
抄送: Zhengzhao Li <zhengzhao.li@uq.net.au>; Author <author@noreply.github.com>
主题: Re: [shakes76/PatternFlow] Topic recognition 45542976 zhengzhgao li (#148)


  *   Some commit messages should be more specific
  *   Cannot locate main training loop for training, test.py is does not have valid imports, results are hence questionable.
  *   Code INVALID for Easy Difficulty

Other Feedback

  *   I would not use "1" as the folder name
  *   Please group the .png files and put them in a sub-directory
  *   Please follow python file and folder naming convention e.g. lorem_ipsum_123_abc.py
  *   Many images not showing

IMPOTANT: YOU MUST JUSTIFY YOUR RESULTS via a reply. I need to know how exactly your code works and how the results came to be.

―

**Comment by SiyuLiu0329** [comment]
You may create another pull request with the CORRECT timestamps.

**Comment by zhengzhao610** [comment]
After the final exam I have cleaned up some files form my laptop, some files have been lost  sorry about that😰

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: SiyuLiu0329 <notifications@github.com>
发送时间: Sunday, November 22, 2020 6:01:58 PM
收件人: shakes76/PatternFlow <PatternFlow@noreply.github.com>
抄送: Zhengzhao Li <zhengzhao.li@uq.net.au>; Author <author@noreply.github.com>
主题: Re: [shakes76/PatternFlow] Topic recognition 45542976 zhengzhgao li (#148)


You may create another pull request with the CORRECT timestamps.

**Comment by zhengzhao610** [comment]
I really prepared this assignment very seriously, just due to laptop issues so some file lost, hope it will not affect my grades plssss🥺

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Zhengzhao Li <zhengzhao.li@uq.net.au>
发送时间: Sunday, November 22, 2020 6:08:37 PM
收件人: shakes76/PatternFlow <reply@reply.github.com>; shakes76/PatternFlow <PatternFlow@noreply.github.com>
抄送: Author <author@noreply.github.com>
主题: Re: [shakes76/PatternFlow] Topic recognition 45542976 zhengzhgao li (#148)

After the final exam I have cleaned up some files form my laptop, some files have been lost  sorry about that😰

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: SiyuLiu0329 <notifications@github.com>
发送时间: Sunday, November 22, 2020 6:01:58 PM
收件人: shakes76/PatternFlow <PatternFlow@noreply.github.com>
抄送: Zhengzhao Li <zhengzhao.li@uq.net.au>; Author <author@noreply.github.com>
主题: Re: [shakes76/PatternFlow] Topic recognition 45542976 zhengzhgao li (#148)


You may create another pull request with the CORRECT timestamps.

**Comment by SiyuLiu0329** [comment]
> I really prepared this assignment very seriously, just due to laptop issues so some file lost, hope it will not affect my grades plssss🥺 获取 Outlook for iOS<https://aka.ms/o0ukef>
> […](#)

It is indeed unfortunate. However, I am unable to verify some of the things you mentioned. Also a big chunk of the marks are given to appropriate version control. Code would not have been lost if it had been done correctly. You may want to contact shakes.

**Comment by zhengzhao610** [comment]
What relevant documents do I need to provide to help you to score，Let me see if i can find it

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: SiyuLiu0329 <notifications@github.com>
发送时间: Sunday, November 22, 2020 6:25:03 PM
收件人: shakes76/PatternFlow <PatternFlow@noreply.github.com>
抄送: Zhengzhao Li <zhengzhao.li@uq.net.au>; Author <author@noreply.github.com>
主题: Re: [shakes76/PatternFlow] Topic recognition 45542976 zhengzhgao li (#148)


I really prepared this assignment very seriously, just due to laptop issues so some file lost, hope it will not affect my grades plssss🥺 获取 Outlook for iOShttps://aka.ms/o0ukef
…

It is indeed unfortunate. However, I am unable to verify some of the things you mentioned. Also a big chunk of the marks are given to appropriate version control. Code would not have been lost if it had been done correctly. You may want to contact shakes.

**Comment by zhengzhao610** [comment]
Can you at least give me a passing grade? I really really prepared this project seriously. It’s just because of the laptop problem.I don’t want to fail this course because this dumb reason
plssss🥺

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: SiyuLiu0329 <notifications@github.com>
发送时间: Sunday, November 22, 2020 6:25:03 PM
收件人: shakes76/PatternFlow <PatternFlow@noreply.github.com>
抄送: Zhengzhao Li <zhengzhao.li@uq.net.au>; Author <author@noreply.github.com>
主题: Re: [shakes76/PatternFlow] Topic recognition 45542976 zhengzhgao li (#148)


I really prepared this assignment very seriously, just due to laptop issues so some file lost, hope it will not affect my grades plssss🥺 获取 Outlook for iOShttps://aka.ms/o0ukef
…

It is indeed unfortunate. However, I am unable to verify some of the things you mentioned. Also a big chunk of the marks are given to appropriate version control. Code would not have been lost if it had been done correctly. You may want to contact shakes.

**Comment by shakes76** [comment]
I will view this request soon and provide feedback on what has been submitted and let you know if there are any issues.

**Comment by zhengzhao610** [comment]
Thank you so much,  Because of my fault, I can accept a low scores, as long as don’t let me fail 🥺

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Shakes <notifications@github.com>
发送时间: Sunday, November 22, 2020 6:42:26 PM
收件人: shakes76/PatternFlow <PatternFlow@noreply.github.com>
抄送: Zhengzhao Li <zhengzhao.li@uq.net.au>; Author <author@noreply.github.com>
主题: Re: [shakes76/PatternFlow] Topic recognition 45542976 zhengzhgao li (#148)


I will view this request soon and provide feedback on what has been submitted and let you know if there are any issues.

File: recognition/1/Unet.py [hunk 1]
```diff
@@ -0,0 +1,221 @@
+import os
```
Comment (shakes76) [inline recognition/1/Unet.py:1]: Code header block should be added for all scripts/source files

File: recognition/1/test.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
+# Setup the optimizer and loss function
```
Comment (shakes76) [inline recognition/1/test.py:1]: No imports, this can't work

**Comment by zhengzhao610** [comment]
I wrote it in jupyter notebook at the beginning and then moved to visual studio code, so the Unet and test files are imported same thing

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Shakes <notifications@github.com>
发送时间: Wednesday, November 25, 2020 9:34:05 AM
收件人: shakes76/PatternFlow <PatternFlow@noreply.github.com>
抄送: Zhengzhao Li <zhengzhao.li@uq.net.au>; Author <author@noreply.github.com>
主题: Re: [shakes76/PatternFlow] Topic recognition 45542976 zhengzhgao li (#148)


@shakes76 commented on this pull request.

________________________________

In recognition/1/test.py<https://github.com/shakes76/PatternFlow/pull/148#discussion_r530007942>:

> @@ -0,0 +1,133 @@
+# Setup the optimizer and loss function


No imports, this can't work

―

**Comment by shakes76** [comment]
## TF Usage
Not good
## Good Practice (Design/Commenting)
OK design and comments
## Algorithm 
Driver Script present, but not functional
Shows Usage & Demo & Visualisation & Data usage, Readme sparse
Module present
## Commit Log
OK commit messages 
Progressive commits not used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
**Feedback required. Fix the test script, rename the folder named 1 to something meaningful.**
Fixing the above will only result in obtaining additional feedback marks according to the assessment sheet.
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

**Comment by zhengzhao610** [comment]
thank you shakes

获取 Outlook for iOS<https://aka.ms/o0ukef>
________________________________
发件人: Shakes <notifications@github.com>
发送时间: Wednesday, November 25, 2020 9:38:48 AM
收件人: shakes76/PatternFlow <PatternFlow@noreply.github.com>
抄送: Zhengzhao Li <zhengzhao.li@uq.net.au>; Author <author@noreply.github.com>
主题: Re: [shakes76/PatternFlow] Topic recognition 45542976 zhengzhgao li (#148)


TF Usage

Not good

Good Practice (Design/Commenting)

OK design and comments

Algorithm

Driver Script present, but not functional
Shows Usage & Demo & Visualisation & Data usage, Readme sparse
Module present

Commit Log

OK commit messages
Progressive commits not used

Documentation

Description and Comments good
Markdown used PDF submitted

Pull Request

Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch)
Feedback required. Fix the test script, rename the folder named 1 to something meaningful.
Fixing the above will only result in obtaining additional feedback marks according to the assessment sheet.
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

―

**Comment by zhengzhao610** [comment]
Hi, Shakes:
I have already changed my folder'name as brain segmentation which is a meaningful name. Besides, some packages were imported into test file. Finally, the pull request is successful.

**Comment by shakes76** [comment]
I will add the feedback marks to your total, but I cannot merge until you fix the remove of my Info-VAE project and broken links in readme for the images.
